packages/ui-kit/src/hooks/useCounter/useCounter.ts

```diff
@@ -48,6 +48,8 @@ export const useCounter = (props: CounterQueryProps): UseQueryProps<CounterQuery
   // Define metric input
   const metricInput = typeof metric === 'string' ? { metricName: metric } : { metric: metric }
 
+  const withTimeRange = timeRange != null ? { ...timeRange } : {}
```

Comment:
instead of this, maybe we can add the props as default like below
```suggestion
  const withTimeRange = timeRange != null ? { ...timeRange } : {}
```

```suggestion
  {
    ...
    timeRange = {},
    ...
  }
  // delete  const withTimeRange = timeRange != null ? { ...timeRange } : {} 

  counterInput: {
    ...
    timeRange,
    ...
  }
```

Reply:
I had to change this a little bit.

If we do this each prop inside `timeRange` would be passed at the top level, e.g `{ relative, n }` instead of `{ timeRange: { relative, n } }`, also we ideally don't pass an empty object for `timeRange`

Reply:
const withTimeRange = timeRange != null ? { ...timeRange } : {}
or
{  time = {} } = props
the same right?

Reply:
const withTimeRange = timeRange != null ? { timeRange: { ...timeRange } } : {}

this change seems good now.